sdk-autoconfigure-support/src/main/java/io/opentelemetry/javaagent/autoconfigure/ResourceProviderPropertiesCustomizer.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.javaagent.tooling.config;
+package io.opentelemetry.javaagent.autoconfigure;
```

Comment:
probably shouldn't be under `io.opentelemetry.javaagent.*` if it's also for use by library instrumentation